scripts/irods/test/test_iadmin.py

```diff
@@ -1629,6 +1629,77 @@ def check_and_remove_rebalance_visibility_metadata(attempts):
         with session.make_session_for_existing_admin() as admin_session:
             admin_session.assert_icommand(['ils', '-l', '-r', self.test_dir], 'STDOUT_SINGLELINE', self.test_dir)
 
+class Test_Iadmin_modzone(session.make_sessions_mixin([('otherrods', 'rods')], [('alice', 'apass')]), unittest.TestCase):
```

Comment:
These tests are actually testing `msiRenameLocalZoneCollection` and should probably be moved to test_all_rules.py.